app/models/user.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
```

Comment:
Don't forget to update the remember_digest to nil when user signs out. You'd need to create a method for that. [OPTIONAL]